core/ecschema-metadata/src/Metadata/Schema.ts

```diff
@@ -522,6 +523,17 @@ export class Schema implements CustomAttributeContainerProps {
     return schemaJson as SchemaProps;
   }
 
+  /**
+   * Serializes the schema to a string of XML
+   * @alpha
+   */
+  public async toXmlString(): Promise<string> {
```

Comment:
We cannot add this here because xmldom is a dev dependency ... we do not want to make it a full dependency